src/main/java/software/sigma/internship/service/impl/TestServiceImpl.java

```diff
@@ -20,36 +20,38 @@
 @AllArgsConstructor
 public class TestServiceImpl implements TestService {
     private final TestRepository testRepository;
-    private final TestMapper testMapper;
-    private final TeacherMapper teacherMapper;
+    private final ModelMapper testMapper;
+    private final ModelMapper teacherMapper;
```

Comment:
I don't think, that you need mapper twice.

Reply:
You are right, I need only one since I use ModelMapper Library